.circleci/config.yml

```diff
@@ -73,10 +89,17 @@ workflows:
   version: 2
   test-deploy:
     jobs:
-      - test
+      - build
+      - test-local:
+          requires:
+            - build
+      - test-bs:
+          requires:
+            - build
```

Comment:
I think we should add `- test-local` here as well?

e.g. we only wanna bother browserstack once the local test succeeded right?

Reply:
I could go both ways. If we don’t block on the bs tests then it should be fine. I _do_ like that running them in parallel like this gets the whole build to less than 5 minutes pretty regularly. Confirm you want to run synchronized and I’ll update.

Reply:
true let's try parallel and if at any point we want to reduce the load on browser stack we can easily do so 🤗